AxonDeepSeg/network_construction.py

```diff
@@ -131,107 +94,87 @@ def uconv_net(x, training_config, phase, bn_updated_decay = None, verbose = True
         bn_decay = bn_updated_decay
 
     # Input picture shape is [batch_size, height, width, number_channels_in] (number_channels_in = 1 for the input layer)
-    net = tf.reshape(x, shape=[-1, image_size, image_size, 1])
-    data_temp = x
+
     data_temp_size = [image_size]
     relu_results = []
 
     ####################################################################
     ######################### CONTRACTION PHASE ########################
     ####################################################################
-    
+
+    # print(data_temp)
+
+    # X = Input((image_size*image_size, 3))
+    X = Input((image_size, image_size, 3))
+
+    net = X
+    data_temp = X
+    # print(net.shape)
+    # print(depth, number_of_convolutions_per_layer)
     for i in range(depth):
 
         for conv_number in range(number_of_convolutions_per_layer[i]):
+
             if verbose:
-                print(('Layer: ', i, ' Conv: ', conv_number, 'Features: ', features_per_convolution[i][conv_number]))
-                print(('Size:', size_of_convolutions_per_layer[i][conv_number]))
-
-            net = conv_relu(net, features_per_convolution[i][conv_number][1], 
-                            size_of_convolutions_per_layer[i][conv_number], k_stride=1, 
-                            w_initializer=tf.contrib.layers.xavier_initializer_conv2d(),
-                            training_phase=phase, activate_bn=activate_bn, bn_decay = bn_decay,
-                            keep_prob=dropout,
-                            scope='cconv-d'+str(i)+'-c'+str(conv_number))
-                
-        relu_results.append(net) # We keep them for the upconvolutions
+                # print(('Layer: ', i, ' Conv: ', conv_number, 'Features: ', features_per_convolution[i][conv_number]))
```

Comment:
We used to print out this information, can we still do so with Keras?

Reply:
(since here, it's in a `verbose` block, which means we expect to have some information displayed if that flag is on.

Reply:
In keras we can view the summary of model using `model.summary()` returned by `train_network.py` which displays all the information of the model.  I will add this in notebook for training in keras. Can we remove verbose block out of the training because I feel it unnecessary?

Reply:
Ok for the first point. For the second, it depends; we'll have to check if it was a command-line option. Let's keep it as-is for now.